content/references/rippled-api/public-rippled-methods/nft-methods/account_nfts.md

```diff
@@ -0,0 +1,118 @@
+---
+html: account_nfts.html
+parent: nft-methods.html
+blurb: Get a list of all NFTs for an account.
+labels:
+  - Non-fungible Tokens, NFTs
+---
+# account_nfts
+[[Source]](https://github.com/ripple/rippled/blob/xls20/src/ripple/rpc/handlers/AccountObjects.cpp "Source")
+{% include '_snippets/nfts-disclaimer.md' %}
+
+The `account_nfts` method returns a list of `NFToken` objects for the specified account.
```

Comment:
As a best practice, the summary sentence should link back to a more general explanation for import keywords like "NFToken objects" and "account" in case people find their way to the API method without the relevant contextual information.

Many of the old API method pages _don't_ do this, which is a known issue (#1024), but we can at least not continue the problem in new pages.